_includes/integrations/react-quick-start.md

```diff
@@ -18,69 +18,65 @@ This procedure requires:
 
 ### Procedure
 
-1. On a command line or command prompt, install the [Create React App](https://github.com/facebook/create-react-app) package.
+1. Use the the [Create React App](https://github.com/facebook/create-react-app) package to create a new React project named `tinymce-react-demo`.
 
     ```sh
-    $ npm install -g create-react-app
+    $ npx create-react-app tinymce-react-demo
     ```
 
-2. Create a new React project named `tinymce-react-demo`.
-
-    ```sh
-    $ create-react-app tinymce-react-demo
-    ```
-
-3. Change into the newly created directory.
+2. Change into the newly created directory.
```

Comment:
Maybe?
```suggestion
2. Navigate into the newly created directory.
```

Reply:
Considering the command is "change directory" I prefer the original...

Reply:
"Change to" then? Rather than "Change into"?

Reply:
done